views/agreement.hbs

```diff
@@ -161,7 +161,7 @@ input[type="submit"] {
 
- <p>An <a href="#individual">Individual</a> or <a href="#entity">Entity</a> may terminate <a href="https://whatwg.org/workstream-policy#workstream-participant">Workstream Participation</a> or this <a href="#agreement">Agreement</a> by providing written notice to the <a href="https://whatwg.org/sg-agreement#steering-group">Steering Group</a>, or the <a href="https://whatwg.org/sg-agreement#steering-group">Steering Group</a> may terminate this <a href="#agreement">Agreement</a> by providing written notice to an <a href="#individual">Individual</a> or <a href="#entity">Entity</a>. Termination is effective upon delivery. With respect to <a href="#individual">Individuals</a> invited by the <a href="https://whatwg.org/sg-agreement#steering-group">Steering Group</a>, <a href="https://whatwg.org/workstream-policy#workstream-participant">Workstream Participation</a> and this <a href="#agreement">Agreement</a> shall otherwise automatically terminate upon the expiration of the <a href="#invited-individual-term">Invited Individual’s Term</a>, unless extended by the <a href="https://whatwg.org/sg-agreement#steering-group">Steering Group</a>. Obligations (including but not limited to licensing obligations) incurred prior to termination do not cease. As a condition of participation in the WHATWG, each <a href="https://whatwg.org/workstream-policy#workstream-participant">Workstream Participant</a> agrees that no claim of any kind may arise from such removal or termination.
+ <p>An <a href="#individual">Individual</a> or <a href="#entity">Entity</a> may terminate <a href="https://whatwg.org/workstream-policy#workstream-participant">Workstream Participation</a> or this <a href="#agreement">Agreement</a> by providing written notice to the <a href="https://whatwg.org/sg-agreement#steering-group">Steering Group</a>, or the <a href="https://whatwg.org/sg-agreement#steering-group">Steering Group</a> may terminate this <a href="#agreement">Agreement</a> by providing written notice to an <a href="#individual">Individual</a> or <a href="#entity">Entity</a>. Termination is effective upon delivery. With respect to <a href="#individual">Individuals</a> invited by the <a href="https://whatwg.org/sg-agreement#steering-group">Steering Group</a>, <a href="https://whatwg.org/workstream-policy#workstream-participant">Workstream Participation</a> and this <a href="#agreement">Agreement</a> shall otherwise automatically terminate upon the expiration of the <a href="https://whatwg.org/invitation-policy#invited-individual">Invited Individual</a> status, unless extended by the <a href="https://whatwg.org/sg-agreement#steering-group">Steering Group</a>. Obligations (including but not limited to licensing obligations) incurred prior to termination do not cease. As a condition of participation in the WHATWG, each <a href="https://whatwg.org/workstream-policy#workstream-participant">Workstream Participant</a> agrees that no claim of any kind may arise from such removal or termination.
```

Comment:
It's hard to spot the difference here, it's "expiration of the Invited Individual status" instead of "expiration of the Invited Individual’s Term"

Reply:
Linking that seems like a good idea 👍

Reply:
Updated. To spot the changes now, start reading at "With respect to"